addons/website_event_sale/tests/test_frontend_buy_tickets.py

```diff
@@ -87,3 +116,67 @@ def test_buy_last_ticket(self):
         self.start_tour("/", 'event_buy_last_ticket')
 
     # TO DO - add public test with new address when convert to web.tour format.
+
+
+class TestUnitPriceUi(TestUiCommon):
+    """Test of the controller used to display the unit price.
+
+    This price depends on the selected quantity and the user groups. The last determines whether the price should be
+    tax included or excluded. """
+    def setUp(self):
+        super().setUp()
+        self.headers_json = {"Content-Type": "application/json", }
+        self.company_currency = self.env.company.currency_id
+        self.precision_delta = self.company_currency.rounding
+
+    def setup_display_tax_incl(self):
+        self.env['res.config.settings'].create({
+            'auth_signup_uninvited': 'b2c',
```

Comment:
Why do you need this?

Reply:
It was inspired from the website_sale test but indeed here no need for free sign up. I have removed it.